src/main.cpp

```diff
@@ -1,6 +1,22 @@
 #include <iostream>
 
 int main(int argc, char *argv[]){
-    std::cout << "Hello,World!" << std::endl;
+    // set a limit of the number of arguments to be passed: in argc
+    if(argc > 4)
+    {
+        // tell the user how to run the program
+        std::cout<<"Usage1: passgen -L length_of_the_password_you_want\n";
+        // We can add other usage messages here based on the input error committed by the user.
+        return 1;
+    }
+
+    // set the length of the character set
+    if(argv[1] == "-L")
```

Comment:
you must convert `argv[1]` to `std::string` as it has overloaded operator `==` for string comparision. If you don't want to convert it to the string object and then compare, you can use C `strcmp` function here which is also faster than converting to `std::string` object.